ext/pcntl/pcntl.c

```diff
@@ -1518,6 +1515,11 @@ PHP_FUNCTION(pcntl_rfork)
     flags &= ~RFMEM;
   }
 
+  if ((flags & (RFFDG | RFCFDG)) != 0) {
+	  zend_argument_value_error(1, "RFFDG and RFCFDG are mutually exclusive");
+	  RETURN_LONG(-1);
```

Comment:
Please use this instead:
```suggestion
	  RETURN_THROWS();
```